src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/DraftApprenticeship/RecognisePriorLearningData.cshtml

```diff
@@ -0,0 +1,128 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.PriorLearningDataViewModel
+@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@{
+    ViewData["Title"] = "Add recognition of prior learning details";
+    ViewBag.ValidationOrder = "TrainingTotalHours,DurationReducedByHours,IsDurationReducedByRpl,ReducedDuration,CostBeforeRpl,ReducedPrice";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+
+        <form method="post">
+            @Html.AntiForgeryToken()
```

Comment:
I believe AntiForgeryTokens are added automatically, so this isn't required.

Reply:
Thanks Chris, resolved locally